rpm/phonesim.spec

```diff
@@ -1,18 +1,8 @@
-# 
-# Do NOT Edit the Auto-generated Part!
-# Generated by: spectacle version 0.27
-# 
-
 Name:       phonesim
-
-# >> macros
-# << macros
-
 Summary:    Phone simulator.
 Version:    1.19.0
 Release:    1
-Group:      Applications/System
-License:    LGPL v2.1
+License:    LGPLv2
```

Comment:
Is this correct? `COPYING` is for `GPLv2` and headers seem to indicate that too. (Also funny enough, headers talk about `LICENSE.GPL` and not `COPYING`).

Reply:
Oh indeed. Changed to GPLv2